Slim/Control/Queries.pm

```diff
@@ -5177,6 +5177,7 @@ my %tagMap = (
 	  'e' => ['album_id',         '',              'albumid'],          #album
 	  'l' => ['album',            'ALBUM',         'albumname'],        #->album.title
 	  'b' => ['work',             'WORK',          'worktitle'],        #->work.title
+	  '1' => ['performance',      'PERFORMANCE',   'performance'],      #performance
```

Comment:
Wow... 9 more options... 🙈 😁.

Please don't forget to update the CLI documentation 😉.

https://github.com/LMS-Community/lms-community.github.io/blob/main/docs/reference/cli/database.md/#songinfo

Reply:
Once this all settles down (there will be more!) I'll update the documents in one go.